integrations/viem-v2/src/index.ts

```diff
@@ -123,9 +123,6 @@ export async function createSapphireSerializer<
 	const fetcher = new KeyFetcher();
 	const provider = client as EthereumProvider;
 	await fetcher.fetch(provider);
-	setTimeout(async () => {
```

Comment:
Lets document the reason for having the interval timer.

It be changed to `setInterval`, so it's continuous.

And the interval ID should be `unref`'d

Reply:
Agree. `unref` seems best right now.

Reply:
Have you been able to reproduce the problem with a test? Would be good to have this in CI